3rdparty/mbedtls/rename-mbedtls-symbols

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Explanation:
+#   readelf -sW: dump symbol table (-s) with the full/wide names (-W).
+#   awk '...': If the symbol is global and begins with mbedtls, then output
+#              the pair $SYMBOL oe_$SYMBOL to feed to objcopy. The seen
+#              array prevents printing duplicates.
+#   objcopy: --redefine-syms will take each $SYMBOL replace it with oe_$SYMBOL.
+readelf -sW "$1" | awk '$5 == "GLOBAL" && $NF ~ /^mbedtls/ && !seen[$NF] {print $NF,"oe_"$NF; seen[$NF]++} {}' > symbols.txt
```

Comment:
As written, this script won't work on Windows. 

1. Is there no plans to support this on Windows?
2. On Windows, we have bash, llvm-objdump. I think llvm also has an llvm-objcopy. Can this script be rewritten to use objdump and objcopy?